lms/services/exceptions.py

```diff
@@ -12,7 +16,7 @@ class LTILaunchVerificationError(ServiceError):
     """
 
 
-class ConsumerKeyError(LTILaunchVerificationError):
+class ConsumerKeyLaunchVerificationError(LTILaunchVerificationError):
     """Raised when a given ``consumer_key`` doesn't exist in the DB."""
```

Comment:
Should this mention that the `consumer_key` in this case is from LTI params to differentiate it from the new exception?